src/Dispatcher/DebugStartupDispatcher.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Dispatcher;
+
+use Psr\Container\ContainerInterface;
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\Yii\Debug\Debugger;
+use Yiisoft\Yii\Web\Event\ApplicationStartup;
+
+class DebugStartupDispatcher implements EventDispatcherInterface
```

Comment:
I think we can just subscribe to the event in question. Are you sure we need to implement dispatcher?

Reply:
This dispatcher starts all collectors. I we can catch the `ApplicationStartup` event before that it will be passed to any collectors then ok. 
So far I have not come up with a more elegant solution. If you know I listen to you.